liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -140,6 +159,74 @@ protected ServerTask initializeJava() {
         serverTask.setOutputDir(outputDirectory);
         return serverTask;
     }
+    
+    protected void runMojo(String groupId, String artifactId, String goal) throws MojoExecutionException {
+        Plugin plugin = getPlugin(groupId, artifactId);
+        Xpp3Dom config = ExecuteMojoUtil.getPluginGoalConfig(plugin, goal, log);
+        log.info("Running " + artifactId + ":" + goal);
+        log.debug("configuration:\n" + config);
+        executeMojo(plugin, goal(goal), config,
+                executionEnvironment(project, session, pluginManager));
+    }
+    
+    /**
+     * Given the groupId and artifactId get the corresponding plugin
+     * 
+     * @param groupId
+     * @param artifactId
+     * @return Plugin
+     */
+    protected Plugin getPlugin(String groupId, String artifactId) {
+        Plugin plugin = project.getPlugin(groupId + ":" + artifactId);
+        if (plugin == null) {
+            plugin = plugin(groupId(groupId), artifactId(artifactId), version("RELEASE"));
+        }
+        return plugin;
+    }
+    
+    protected Plugin getLibertyPlugin() {
+        Plugin plugin = project.getPlugin(LIBERTY_MAVEN_PLUGIN_GROUP_ID + ":" + LIBERTY_MAVEN_PLUGIN_ARTIFACT_ID);
+        if (plugin == null) {
+            plugin = plugin(LIBERTY_MAVEN_PLUGIN_GROUP_ID, LIBERTY_MAVEN_PLUGIN_ARTIFACT_ID, "LATEST");
```

Comment:
And LATEST is deprecated too, right?

Reply:
This code was unchanged, just moved to a new location. I had discussed using `LATEST` and `RELEASE` before with Eric and we had agreed that it was ok for the time being while we figure out a better solution.